packages/components/CHANGELOG.md

```diff
@@ -4,10 +4,13 @@
 
 ### Enhancements
 
+-   Export `IconType` type for external use. ([#49649](https://github.com/WordPress/gutenberg/pull/49649))
+
+### Enhancements
+
 -   `TreeGrid`: Modify keyboard navigation code to use a data-expanded attribute if aria-expanded is to be controlled outside of the TreeGrid component ([#48461](https://github.com/WordPress/gutenberg/pull/48461)).
 -   `Modal`: Equalize internal spacing ([#49890](https://github.com/WordPress/gutenberg/pull/49890)).
 -   `Modal`: Increased border radius ([#49870](https://github.com/WordPress/gutenberg/pull/49870)).
--   `SlotFill`: Added util for creating private SlotFills and supporting Symbol keys ([#49819](https://github.com/WordPress/gutenberg/pull/49819)).
```

Comment:
This looks like it might have been caused by a bad rebase. We should likely revert it.